src/main/java/de/tum/cit/aet/artemis/hyperion/proto/hyperion.proto

```
@@ -0,0 +1,138 @@
+syntax = "proto3";
+
+package de.tum.cit.aet.artemis.hyperion;
```

Buf (1.54.0)

Files with package "de.tum.cit.aet.artemis.hyperion" must be within a directory "de/tum/cit/aet/artemis/hyperion" relative to root but were in directory "src/main/java/de/tum/cit/aet/artemis/hyperion/proto".

(PACKAGE_DIRECTORY_MATCH)